src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -79,7 +79,26 @@ public static AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    /**
+     * Returns an {@code AddressBook} with all the typical persons.
+     */
+    public static AddressBook getUnTypicalAddressBook() {
```

Comment:
UnTypicalAddressBook??

(I gather you mean unsorted, but please specify in both the function name and the javadoc!)

Reply:
done